README.md

```diff
@@ -6,6 +6,7 @@
 1. Move `.env-sample` to `.env` and replace `TOKEN` (ask @benheller)
 1. `nvm install` <-- select node version
 1. `yarn install` <-- install deps
+1.  E.g. `sudo service mongod start` if running mongo locally
```

Comment:
Do we need the `E.g` here, since the other commands don't have this prefix? Also, `sudo service mongod start` is specific to particular flavors of Unix (others might use `systemctl`). I like that you included it in the MongoDB section below, so maybe remove it here, and add a header to make clear it's the Linux instruction? If you want to add it for Mac too, the command would be `brew service start mongodb-community`

Reply:
This line is already in the README. I guess it's in the PR because of some messy branching.

It says "E.g" BECAUSE it's specific to a particular flavor of Unix.

I don't think you need my permission to edit the README to your liking. I needed something there to remind me to start Mongo.